src/components/DialoguesList/DialoguesList.jsx

```diff
@@ -7,7 +7,7 @@ import MessageBubble from '../MessageBubble/MessageBubble';
 
 /**
  * DialoguesList component displays messages in the form of MessageBubbles between users.
- * This component takes in an array of objects (messages) which contain properties sender id, text, 
+ * This component takes in an array of objects (messages) which contain properties sender id, text,
```

Comment:
Shouldn't this comment be in the same line?

Reply:
I'll fix the comment as it's not marked as done yet :)